src/eddymotion/registration/ants.py

```diff
@@ -86,6 +110,217 @@ def _prepare_registration_data(dwframe, predicted, affine, vol_idx, dirname, reg
     return fixed, moving
 
 
+def _get_ants_settings(settings="b0-to-b0_level0"):
+    return Path(
+        pkg_fn(
+            "eddymotion.registration",
+            f"config/{settings}.json",
+        )
+    )
+
+
+def _massage_mask_path(mask_path, nlevels):
+    """
+    Generate nipype-compatible masks paths.
+
+    Examples
+    --------
+    >>> _massage_mask_path("/some/path", 2)
+    ['/some/path', '/some/path']
+
+    >>> _massage_mask_path(["/some/path"] * 2, 2)
+    ['/some/path', '/some/path']
+
+    >>> _massage_mask_path(["/some/path"] * 2, 4)
+    ['NULL', 'NULL', '/some/path', '/some/path']
+
+    >>> _massage_mask_path(["/some/path"] * 2, 1)
+    ['/some/path']
+
+    """
+    if isinstance(mask_path, (str, Path)):
+        return [str(mask_path)] * nlevels
+    if len(mask_path) < nlevels:
+        return ["NULL"] * (nlevels - len(mask_path)) + mask_path
+    if len(mask_path) > nlevels:
+        warn("More mask paths than levels", stacklevel=1)
+        return mask_path[:nlevels]
+    return mask_path
+
+
+def generate_command(
+    fixed_path,
+    moving_path,
+    fixedmask_path=None,
+    movingmask_path=None,
+    init_affine=None,
+    default="b0-to-b0_level0",
+    **kwargs,
+):
+    """
+    Generate an ANTs' command line.
```

Comment:
Not a show stopper at this stage, but we may be willing to add parameter docstrings for this method at some point.